src/lib/party.ts

```diff
@@ -782,3 +801,78 @@ export class Party<P> {
     return party;
   }
 }
+
+function toParty(party: Party): Party_ {
```

Comment:
It's scary that we have this subtly different `Party` and `Party_` on the bridge. We've had similar weird glue in parts of the Mina code that led to pain which eventually we threw out. If it's easy enough, can we unify these now? If it's not easy enough or there are design questions to consider, can you make an issue and then we can triage it when we sort polish tasks? Or if you think we shouldn't unify these, feel free to explain in a reply here.

Reply:
I agree! But first, I want to auto-generate some of the TS types in snarky.d.ts, i.e. `Party_` -- see https://github.com/MinaProtocol/mina/issues/10553
After that, I definitely want to unify these two type systems. It's crazy how the type conversions I have to do continue to expand. It's the first thing I want to tackle once all the Berkeley interactions are working end-to-end (which is almost the case)

Reply:
created issue https://github.com/o1-labs/snarkyjs/issues/132